src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -376,6 +425,34 @@ bool CascadiaSettings::_IsPowerShellCoreInstalled(std::wstring_view programFileE
     return false;
 }
 
+// Function Description:
+// - Returns true if the executable for the distribution was found.
+// Arguments:
+// - A string that contains the distributions directory.
+// - A string representing the distinct WSL distribution.
+// - A ref of a path that receives the result of [distribution].exe's full path.
+// Return Value:
+// - true or false.
+bool CascadiaSettings::_IsWSLDistributionAvailable(std::filesystem::path WSLDirectory, std::wstring searchString, std::filesystem::path& cmdline)
+{
+    // step through all of the directory names to find the one with our search string
+    for (auto& directory : std::filesystem::directory_iterator(WSLDirectory)) {

```

Comment:
nit: I believe Terminal uses Allman-style braces; see #609.

Reply:
Also, prefer const auto& where possible to clearly express intent [per ISO C++ guideline P3](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#p3-express-intent).

Reply:
Regarding const auto&, the PowerShell function above is also lacking it.  I think this should be included in a modernization or clarification PR, so I left it out of my changes.